subcommand/injector/flags.go

```diff
@@ -13,8 +13,10 @@ import (
 )
 
 const (
-	DefaultLogLevel  = "info"
-	DefaultLogFormat = "standard"
+	DefaultLogLevel                    = "info"
+	DefaultLogFormat                   = "standard"
+	defaultTLSMinVersion               = "tls12"
+	defaultTLSPreferServerCipherSuites = true
```

Comment:
👍

Reply:
Is this flag/option needed? Reading from https://pkg.go.dev/crypto/tls#Config, it looks like this deprecated and is now completely ignored.

Reply:
Right, we can remove the option if we update our builds to go 1.17; they're on 1.16 currently.

Reply:
main is now building with go 1.17. I merged that into this branch, and removed the `tls_prefer_server_cipher_suites` option.